lms/djangoapps/support/docs/decisions/0001-sso-history-in-support.rst

```diff
@@ -0,0 +1,34 @@
+1. Registering SSO History Model in Support
+================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+SSO History was one of the feature requested by support that provides the 
+historical data of any particular SSO record (based off UserSocialAuth)
```

Comment:
nit: either use `based off of` or `based on`

Reply:
Fixed